stratum/hal/bin/barefoot/BUILD

```diff
@@ -129,5 +181,28 @@ pkg_deb(
     postinst = "deb/postinst",
     prerm = "deb/prerm",
     recommends = ["systemd"],
-    version = "0.0.1",
+    version = "0.0.2",
```

Comment:
Why increment the number? We don't plan to increment it again, making the `--reinstall` mandatory anyway.

Reply:
I guess we can roll this back, but in the spirit of a new release, this would allow people to avoid the `--reinstall`